test/acceptance-tests/use-cases/test-scripts/create-model-config.js

```diff
@@ -69,7 +71,6 @@ module.exports = function() {
   this.Then(/^the model config is returned$/, function(next) {
     expect(Object.keys(testsuite.modelConfig)).to.include.members([
```

Comment:
check for the single key instead

`expect(testsuite.modelConfig).to.have.key('dataSource');`